downstairs/src/lib.rs

```diff
@@ -2836,6 +2816,13 @@ impl Downstairs {
         Ok(())
     }
 
+    /// See the comment in the proc() function on the upstairs side that
```

Comment:
This is good, but we don't have `proc()` in the upstairs any longer :)
I think this should be the method: `continue_negotiation()`

Reply:
Fixed!